File: Copilot/1. Algoritmo Básico Ordenamiento por Burbuja/Javascript/burbuja.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Ingresa números separados por espacio: ", (input) => {
    let arr = input.split(" ").map(Number);

    for (let i = 0; i < arr.length; i++) {
        for (let j = 0; j < arr.length - i - 1; j++) {
            if (arr[j] > arr[j + 1]) {
                [arr[j], arr[j + 1]] = [arr[j + 1], arr[j]];
            }
        }
    }

    console.log("Lista ordenada:", arr);
    rl.close();
});

File: Copilot/1. Algoritmo Básico Ordenamiento por Burbuja/Ruby/burbuja.rb
puts "Ingresa números separados por espacio:"
arr gets.chomp.split.map(&:to_i)

(arr.length - 1).times do |i|
  (arr.length - i - 1).times do |j|
    arr[j], arr[j+1] = arr[j+1], arr[j] if arr[j] > arr[j+1]
  end
end

puts "Lista ordenada: #{arr.join(' ')}"
